lib/modulesync/constants.rb

```diff
@@ -6,5 +6,6 @@ module Constants
     MODULESYNC_CONF_FILE = 'modulesync.yml'
     PROJ_ROOT            = './modules'
     HOOK_FILE            = '.git/hooks/pre-push'
+    GLOBAL_DEFAULTS_KEY  = :global
```

Comment:
Pondering collisions more, It might also make sense to NOT merge the global section into `@configs` and put it into a separate member of `ForgeModuleFile`.